klogr/klogr_test.go

```diff
@@ -41,7 +41,7 @@ func testOutput(t *testing.T, format string) {
 			klogr:         new().V(0),
 			text:          "test",
 			keysAndValues: []interface{}{"akey", "avalue"},
-			expectedOutput: ` "msg"="test" "akey"="avalue"
+			expectedOutput: `"msg"="test" "akey"="avalue"
```

Comment:
I like that this empty space at the start is going away.

Reply:
Yes. That space was the result of some overly simplistic code for injecting the prefix, which is now gone.